bin/lesson_check.py

```diff
@@ -342,7 +342,7 @@ def check_line_lengths(self):
                 n > MAX_LINE_LEN) and (not l.startswith('!'))]
             self.reporter.check(not over,
                                 self.filename,
-                                'Line(s) are too long: {0}',
+                                'Line(s) too long: {0}',
```

Comment:
Alternatively, we could expand this to `Line is / Lines are ...` but IMHO this is sufficient to avoid the inconsistency `Line(s) are ...: 1`